src/ImageSharp/Processing/Processors/Convolution/Convolution2DProcessor.cs

```diff
@@ -56,17 +57,19 @@ protected override void OnApply(ImageBase<TPixel> source, Rectangle sourceRectan
             int maxY = endY - 1;
             int maxX = endX - 1;
 
-            using (PixelAccessor<TPixel> targetPixels = new PixelAccessor<TPixel>(source.Width, source.Height))
-            using (PixelAccessor<TPixel> sourcePixels = source.Lock())
+            using (var targetPixels = new PixelAccessor<TPixel>(source.Width, source.Height))
```

Comment:
Yeah, there's a few places I'd like to visit in the codebase re `PixelAccessor`. 

e.g
https://github.com/JimBobSquarePants/ImageSharp/blob/631545a7561817fb7aff2defb4422dbff41c96d6/src/ImageSharp/Formats/Gif/GifDecoderCore.cs#L477

But I think I'll leave that till after this PR. We've touched a lot of files already and I'd like to get the public API out there first.